src/index.js

```diff
@@ -1,2 +1,43 @@
-/* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
+/* eslint-enable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+    const element = document.createElement(tagName);
+
+    Object.entries(props || {}).forEach(([key, value]) => {
+        element[key.toLowerCase()] = value;
+    })
+
+    children.forEach((child) => {
+        if (child instanceof Node) {
+            element.appendChild(child);
+            return;
+        }
+        element.appendChild(document.createTextNode(child));
+    })
+
+    return element;
+}
+
+function handleClick(count = 1) {    
+    render(count);
+}
```

Comment:
저는 어쩔수 없는 경우가 아니라면 모든 로직을 handleClick 함수 안에서 하는걸 좋아합니다! 민재님께서는 어떠신가요?

```suggestion
function handleClick(count = 1) {    
    render(count + 1);
}
```